Trash icon for users and group permission deletions

AlexandruValeanu opened the pull request:
Should close https://github.com/ToolJet/ToolJet/issues/985.

https://user-images.githubusercontent.com/7942531/137081612-43d9cce6-5d5c-4f09-b1f2-54dd999c9134.mp4

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=993) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=993) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ToolJet/ToolJet?pullRequest=993) it.</sub>

AlexandruValeanu commented:
I somehow managed to read that issue wrong multiple times, @Navaneeth-pk. I fixed this PR, added a screen capture and I'll raise another issue for my suggestions.
